src/Notes.h

```diff
@@ -109,12 +109,13 @@ class Notes : public RefCountable
 {
 public:
     typedef RefCount<Notes> Pointer;
+    typedef std::vector<SBuf> KeysList;
     typedef std::vector<Note::Pointer> NotesList;
     typedef NotesList::iterator iterator; ///< iterates over the notes list
     typedef NotesList::const_iterator const_iterator; ///< iterates over the notes list
 
-    Notes(const char *aDescr, const char **metasBlacklist, bool allowFormatted = true): descr(aDescr), blacklisted(metasBlacklist), formattedValues(allowFormatted) {}
-    Notes(): descr(nullptr), blacklisted(nullptr), formattedValues(false) {}
+    explicit Notes(const char *aDescr, const KeysList *metasBlacklist = nullptr, bool allowFormatted = true);
+    explicit Notes();
```

Comment:
Since you are re-arranging the constructors you may as well convert this class to C++11 initialization and use "= default" here instead of adding a new stub file.

Reply:
If I remember correctly, I got strange results when I tried to initialize  Notes::blacklist with the static Notes::BlackList member  as C++11 class member , I think when the Adaptation::Config::metaHeaders initialized inside adaptation/Config.cc file. 
No compile errors but the Notes::blacklist member appeared to have strange  contains. I did not check thoroughly I just reverted to the classic constructor. I considered it as initialization order problem but now I am thinking that maybe the problem was just (local) linking problems.

Do you believe it is important to check and use C++11 here?

Reply:
@chtsanti, It sounds like you experiences the static initialization order fiasco: In general, one should not initialize static Y in y.cc using static X in x.cc. The code will compile but may not work in some builds because the initialization code in y.cc may run _before_ the initialization code in x.cc.

I am not sure, but AFAICT, a similar initialization bug was still in the PR code. It may became hidden from your tests after you shoveled the code around, but I saw no guarantee that the Adaptation::Config::metaHeaders global (that uses Notes) would always be initialized _after_ the Notes::BlackList global (that Notes use).

I fixed that in branch commit b00ac92.

Reply:
@yadij, I got rid of stub_Notes.cc in branch commit d5fa335, hopefully addressing your change request.

Reply:
Yes that removes the issue I had and more.